.rubocop.yml

```diff
@@ -31,6 +30,8 @@ Style/DefWithParentheses:
   Enabled: false
 Style/FrozenStringLiteralComment:
   EnforcedStyle: never
+Style/CaseEquality:
+  Enabled: false
```

Comment:
Let's not change the config file, please.

You can, however, disable it in the source code.

> - [ ] can have `Style/CaseEquality` cop disabled as described here https://rubydoc.info/gems/rubocop/0.24.1/frames#Disabling_Cops_within_Source_Code ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/style_case_equality.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/57421589/))

Reply:
Thanks, got it and fixed.